linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_encrypt_sent_records/oval/shared.xml

```diff
@@ -5,24 +5,36 @@
       <affected family="unix">
         <platform>multi_platform_all</platform>
       </affected>
+{{% if product in ["rhel8", "fedora"] %}}
+      <description>transport setting in /etc/audit/audisp-remote.conf is set to 'KRB5'</description>
+{{% else %}}
       <description>enable_krb5 setting in /etc/audisp/audisp-remote.conf is set to 'yes'</description>
+{{% endif %}}
     </metadata>
 
     <criteria>
-      <criterion comment="enable_krb5 setting in audisp-remote.conf" test_ref="test_auditd_audispd_encrypt_sent_records" />
+      <criterion comment="setting in audisp-remote.conf" test_ref="test_auditd_audispd_encrypt_sent_records" />
```

Comment:
Since `audisp-remote` is not a common element for all products, the comment should be about enabling encryption for transport of audit records.